app/policies/comment_policy.rb

```diff
@@ -78,4 +82,8 @@ def user_is_blocked?
   def user_is_commentable_author?
     record.commentable.present? && record.commentable.user_id == user.id
   end
+
+  def user_is_admin?
+    user.any_admin? || user.tech_admin?
+  end
```

Comment:
Let's not do tech admin for this, since tech admin is for viewing our technical services.